langmeta/js/src/main/scala/java/io/File.scala

```diff
@@ -31,6 +31,10 @@ class File(path: String) {
     toPath.toAbsolutePath.toFile
   def getAbsolutePath: String =
     getAbsoluteFile.toString
+  def getParentFile: File =
+    toPath.getParent.toFile
+  def mkdirs(): Unit =
+    throw new UnsupportedOperationException("mkdirs() is not supported in Scala.js")
```

Comment:
Why do we need these changes?

Reply:
Because langmeta/io is now in the same project as langmeta/semanticdb that used these methods. Previously, it compiled because java.io is on the boot classpath, now it failed because java.io is compiled in the same project. It didn't fail because it got DCEd.